source/issues-and-help.txt

```diff
@@ -0,0 +1,43 @@
+.. _mongodb-analyzer-issues-and-help:
+
+=============
+Issues & Help
+=============
+
+.. default-domain:: mongodb
+
+Often, the quickest way to get support for general questions is through the
```

Comment:
Suggestion: I think [the Go Driver introduction](https://www.mongodb.com/docs/drivers/go/current/issues-and-help/) is stronger and that the intro here should resemble it